### PR TITLE
feat(banner): shows welcome banner for users with 0 activity

### PR DESCRIPTION
**Pull request by seferturan**
## 🎶 Notes 🎶

- Adds welcome banner for users with 0 activity.
- Banner is dismissable.

## 👀 Examples 👀
Screenshot 2026-07-02 at 17 07 57

Screenshot 2026-07-02 at 17 09 16

### CODE REVIEW

File: projects/client/src/lib/sections/banner/welcome/WelcomeBanner.svelte [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      <Button
+        href={UrlBuilder.settings.data()}
+        color="purple"
+        variant="primary"
+        style="flat"
+        size="small"
+        label={m.welcome_banner_action()}
+      >
+        {m.welcome_banner_action()}
+      </Button>
```
[the remark this answers is not in the document]
Comment (seferturan) [reply projects/client/src/lib/sections/banner/welcome/WelcomeBanner.svelte:53]: @gemini-code-assist, no. The children one is used as the button text.

File: projects/client/src/lib/sections/banner/welcome/_internal/useWelcomeBanner.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import type { UserLimits } from '$lib/requests/models/UserLimits.ts';
+import { combineLatest, map } from 'rxjs';
+import { useBannerDismissal } from '../../_internal/useBannerDismissal.ts';
+import { WELCOME_BANNER_ID } from '../constants/index.ts';
+
+const DISMISSAL_VALUE = 'dismissed';
+
+function hasNoActivity(limits: UserLimits): boolean {
+  return Object.values(limits).every((metric) => metric.current === 0);
+}
```
[the remark this answers is not in the document]
Comment (seferturan) [reply projects/client/src/lib/sections/banner/welcome/_internal/useWelcomeBanner.ts:12]: @gemini-code-assist Good catch. Added a `metrics.length > 0` guard. Zod already forces all 8 keys so `{}` shouldn't reach here in practice, but the guard is cheap and makes the intent explicit.

File: projects/client/src/lib/sections/banner/welcome/WelcomeBanner.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (seferturan) [reply projects/client/src/lib/sections/banner/welcome/WelcomeBanner.svelte:53]: @gemini-code-assist `label` is the Button's own prop (drives the accessible label) and the children are the visible text. Passing both with the same message is the established pattern across the app (WelcomeIntro, WelcomeImport, ImporterCard), so keeping it consistent here. Leaving as-is.

**Comment by seferturan** [comment]
@gemini-code-assist addressed the `hasNoActivity` empty-limits edge case with a `metrics.length > 0` guard. The Button `label`+children stays as-is (matches the app-wide pattern). Pushed in b83e1e3, tests green.
